io.github.wdaniau.fstl.yml

```diff
@@ -0,0 +1,25 @@
+id: io.github.wdaniau.fstl
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+command: fstl-e
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=home
```

Comment:
`home` includes `xdg-documents`.

Reply:
noted, I'll suppress xdg-documents

Reply:
it doesn't use portals? Since it's Qt based...

Reply:
I must admit I don't know ... ^^

Reply:
Ok after reading a bit. The only portals that are used in this app are the one described as transparently used on this page : https://docs.flatpak.org/en/latest/portals.html#portal-support-in-qt-and-kde